whiskey/public/data/whiskies.json

```diff
@@ -3171,6 +3171,136 @@
         "Sweet"
       ],
       "average_rating": 71
-    }
-  ]
+    },
+  {
```

Comment:
mis-aligned. Generally, just ensure that you've run the JSON through a validator like this one jsonformatter.curiousconcept.com, so it doesn't break the whole site.